.travis.yml

```diff
@@ -31,9 +31,9 @@ install:
   - travis_retry pip install --upgrade tox tox-travis coveralls codecov
 
 script:
-  - tox
+  - tox -c tox-travis.ini
 
 after_success:
   - travis_retry coveralls --rcfile=tools/coverage_unit_tests.rc
-  - travis_retry codecov
+  - travis_retry codecov --flags unittests
```

Comment:
Where is this flag defined? Everywhere else we refer to unit tests as `unit_tests`.

Reply:
This is used as a tag on codecov (though flags are temporarily disabled there while refactoring). They do not allow using a `_` in a flag for some reason.